conviva/src/main/java/com/bitmovin/analytics/conviva/ssai/SsaiApi.java

```diff
@@ -0,0 +1,168 @@
+package com.bitmovin.analytics.conviva.ssai;
+
+import java.util.Map;
+
+/**
+ * Enables reporting of server-side ad breaks and ads.
+ */
+public interface SsaiApi {
+    /**
+     * Checks if a server-side ad break is currently active.
+     *
+     * @return <code>true</code> if a server-side ad break is active, <code>false</code> otherwise.
+     */
+    boolean isAdBreakActive();
+
+    /**
+     * Reports the start of a server-side ad break. Must be called before the first ad starts.
+     * Has no effect if a server-side ad break is already playing.
+     */
+    void reportAdBreakStarted();
+
+    /**
+     * Reports the start of a server-side ad break. Must be called before the first ad starts.
+     * Has no effect if a server-side ad break is already playing.
+     *
+     * @param adBreakInfo Map containing metadata about the server-side ad break. Can be <code>null</code>.
+     */
+    void reportAdBreakStarted(Map<String, Object> adBreakInfo);
+
+    /**
+     * Reports the end of a server-side ad break. Must be called after the last ad has finished.
+     * Has no effect if no server-side ad break is currently active.
+     */
+    void reportAdBreakFinished();
+
+    /**
+     * Reports the start of a server-side ad.
+     *
+     * @param adInfo Object containing metadata about the server-side ad.
+     */
```

Comment:
Has to be called after `reportAdBreakStarted`, right?

Reply:
Yes, I added it to the documentation.

Reply:
[9a52f65](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/72/commits/9a52f6526daf245ee6f81cb8200664934a101409)